tasks/pkg-redhat/install-apm-inject.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Set APM injection languages and versions for Redhat
```

Comment:
Ditto with locking packages (done in install_script [here](https://github.com/DataDog/agent-linux-install-script/commit/e7aa1679fecaaf76d12f9cf0cfbb07bca19065be#diff-79ac74a50e14d4fa9c80b1daba8bd5cc182f4b6c2a2b3951c0caa4fbc78f6f73R889))

Reply:
Done

Reply:
@bkabrda I am duplicating the function to parse the agent libraries to install here for dnf because i am not sure if this will run in the other call at line 82 of tasks/pkg-redhat.yml. Let me know if that is correct

Reply:
I think it is, but I'll test locally to be 100 % sure. Nice job!